src/node_main.cc

```diff
@@ -27,13 +27,19 @@
 #include <VersionHelpers.h>
 #include <WinError.h>
 
+#define SKIP_CHECK_VAR "NODE_SKIP_PLATFORM_CHECK"
+
 int wmain(int argc, wchar_t* wargv[]) {
   // Windows Server 2012 (not R2) is supported until 10/10/2023, so we allow it
   // to run in the experimental support tier.
   if (!IsWindows8Point1OrGreater() &&
-      !(IsWindowsServer() && IsWindows8OrGreater())) {
+      !(IsWindowsServer() && IsWindows8OrGreater()) &&
+      GetEnvironmentVariableA(SKIP_CHECK_VAR, nullptr, 0) == 0) {
     fprintf(stderr, "This application is only supported on Windows 8.1, "
-                    "Windows Server 2012 R2, or higher.");
+                    "Windows Server 2012 R2, or higher.\n"
+                    "If the environment varaible " SKIP_CHECK_VAR
+                    " is defined this check is skipped, but Node.js might "
+                    "not execute correctly.");
```

Comment:
I would make this wording stronger. Perhaps something like:

```
Setting the NODE_SKIP_PLATFORM_CHECK environment variable skips this
check, but Node.js might not execute correctly. Any issues encountered
on unsupported platforms will not be fixed.
```

Or something along those lines. Just need to make it more explicit that this is running Node.js in an unsupported configuration.